docs/how_to/debug_guide/Introduction.md

```diff
@@ -0,0 +1,21 @@
+**AMD Instinct GPU Troubleshooting Guide**
+
+
+## **Introduction** 
+
+1.  **Overview**
+
+This document is intended to be used by Data Center customers using production servers with Advanced Micro Devices, Inc. Instinct GPUs, to help them debug and troubleshoot some of the common issues encountered during setup, installation and production use of the servers.
```

Comment:
The documentation is general and for all users. None of the information here is specific to Instinct GPUs.